bundle/src/test/java/com/adobe/acs/commons/wcm/impl/PropertyMergePostProcessorTest.java

```diff
@@ -189,6 +200,75 @@ public void testMerge_Duplicates_Calendar() throws Exception {
         Assert.assertArrayEquals(new Calendar[]{ january, september, july, september },
                 properties.get("dates", Calendar[].class));
     }
+    
+    @Test
+    public void testTagDetection() {
+        Assert.assertTrue("Valid root tag detection", PropertyMergePostProcessor.looksLikeTag("Some_Root:Tag"));
```

Comment:
`Assert.assertTrue("Valid tag namspace detection", PropertyMergePostProcessor.looksLikeTag("Some_Root:"));` Should pass as well.